src/app/pages/impact-dev/map/schoolinfo-head.js

```diff
@@ -0,0 +1,22 @@
+import VERSION from '~/version';
+import {Controller} from 'superb.js';
+import $ from '~/helpers/$';
+import {description as template} from './schoolinfo-head.html';
+
+
+export default class SchoolinfoHead extends Controller {
+
+    init(props) {
+        this.template = template;
+        this.css = `/app/pages/impact-dev/map/map.css?${VERSION}`;
+        this.view = {
+            classes: ['list-data', 'detail-info-head']
+        };
+        this.model= props;
+    }
+
+    onLoaded() {
+        $.insertHtml(this.el, this.model);
```

Comment:
no, some items are inserting from model.

Reply:
There are no elements with `data-html` attributes, so `insertHtml` is not doing anything. Normal items interpolated into the view (like `{model.someVariable}`) do not need `insertHtml`. It is only when you have stuff that needs to be interpreted as HTML markup that you use `insertHtml`.
You might have a tag like 
`<h3 class="title" data-html="heading"></h3>`
and an item in model like
`heading: 'Some <b>bold</b> text'`
and `insertHtml` would insert it as HTML (bolding the word bold) rather than as plain text.

Reply:
ok, now i got it.
thaks alot.